sr_port/hd.mpt

```diff
@@ -1,6 +1,9 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;								;
-;	Copyright 1987,2001 Sanchez Computer Associates, Inc.	;
+; Copyright 1987,2001 Sanchez Computer Associates, Inc.		;
```

Comment:
Just a matter of esthetics, but the semi-colon box doesn't line up, at least on the Github review.

Reply:
That is likely because I have replaced the end of the line with tabs (instead of spaces) and that is how all lines in the rest of the copyright notice are. I do see all lines aligned in the actual code.

Reply:
OK